src/TMap.cpp

```diff
@@ -1741,44 +1741,6 @@ bool TMap::retrieveMapFileStats( QString profile, QString * latestFileName = 0,
         if( areaCount ) {
             * areaCount = areaSize;
         }
-        // read each area
```

Comment:
**WHAT IS GOING ON HERE! This is _needed_ to parse through the other profiles's map file!**

Is it a mis-cherry-pick/mis-meld?